src/codegen/cpu.rs

```diff
@@ -83,6 +83,7 @@ impl<'a, Tgt: Target<Level = CpuMemoryLevel>> CpuCodeGenerator<'a, Tgt> {
         writeln!(main_body_str, "}}")?;
 
         self.headers.emit(Tgt::target_id(), out)?;
+        out.write_str("\n")?;
```

Comment:
This is for the newline between headers and the kernel.

Reply:
Thanks! If you break this into another PR, happy to merge into main immediately.